### PR TITLE
test: use IPv6 link-local DNS server

### PR DESCRIPTION
**Pull request by mkowalski**

### CODE REVIEW

File: test/e2e/handler/rollback_test.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ interfaces:
     dhcp: true
     autoconf: true
     enabled: true
-`, nic))
+`, nic, nic, nic))
```
Comment (qinqon) [inline test/e2e/handler/rollback_test.go:95]: can se use golang expression `%[1]s` so we don't have to repeat nic ?

**Review by qinqon — CHANGES_REQUESTED** [review]
nit

**Comment by mkowalski** [comment]
/close

We will merge upstream first and then cherry-pick

File: test/e2e/handler/rollback_test.go (hunk 1 above)
Comment (mkowalski) [reply test/e2e/handler/rollback_test.go:95]: Done in https://github.com/nmstate/kubernetes-nmstate/pull/1225/files

**Pull request closed without merging**